x-pack/plugins/ml/public/application/explorer/reducers/explorer_reducer/get_index_pattern.ts

```diff
@@ -6,14 +6,15 @@
  */
 
 import { ML_RESULTS_INDEX_PATTERN } from '../../../../../common/constants/index_patterns';
+import type { ExplorerJob } from '../../explorer_utils';
 
 // Creates index pattern in the format expected by the kuery bar/kuery autocomplete provider
 // Field objects required fields: name, type, aggregatable, searchable
-export function getIndexPattern(influencers: string[]) {
+export function getIndexPattern(influencers: ExplorerJob[]) {
```

Comment:
It was typed incorrectly, the arg passed here was always of type 'ExplorerJob[]' instead of just 'string[]'. 
The `IndexPattern` in `ExplorerState` type expects a name as a string, but after mapping, it was always an object of type `ExplorerJob`. I didn't notice any regressions after this change, but I would appreciate testing to ensure everything works correctly.